package.json

```
@@ -26,7 +26,7 @@
     "lint": "tsc --noEmit && eslint .",
     "prepare": "husky install",
     "pre-commit": "lint-staged",
-    "test": "jest --runInBand --testTimeout=60000",
+    "test:unit": "jest --runInBand --testTimeout=60000",
```

Minor nit: I have a slight preference to keeping the old one in addition for a short time, but we're a small team so adjusting to the new name probably won't take long either. I'll leave it to you.

```suggestion
    "test": "jest --runInBand --testTimeout=60000",
    "test:unit": "jest --runInBand --testTimeout=60000",
```